usco/Jam
# Mirror tool

Add the ability to mirror parts along x,y,z

## Checklist

- [x] add basic mirror support
- [x] add ui to set mirroring axis 

## How to test
- Open Jam with a model
- select a model
- click on the mirroring tool
- click on which axis you want to mirror along 
- the part should now be mirrored along the axis chosen above
- the normals should not be inverted ('inside out look')
